README.md

```diff
@@ -40,7 +36,11 @@ interface IObserverOptions {
 
 The observer converts a component into a reactive component, which tracks which observables are used automatically and re-renders the component when one of these values changes.
 
-### **`useObserver<T>(fn: () => T, baseComponentName = "observed", options?: IUseObserverOptions): T`** _([user guide](https://mobx-react.js.org/observer-hook))_
+### **`<Observer>{renderFn}</Observer>`**
+
+Is a React component, which applies observer to an anonymous region in your component.
+
+### **`useObserver<T>(fn: () => T, baseComponentName = "observed", options?: IUseObserverOptions): T`**
```

Comment:
Remove this from here since it's no longer a public API?

Reply:
yeah didn't touch the readme.

Reply:
Updated the readme coursely